README.md

```diff
@@ -107,6 +107,23 @@ Then, open `chrome://inspect` in Chrome to attach the debugger (look into the "R
 
 This project is set up to use [jest](https://facebook.github.io/jest/) for tests. You can configure whatever testing strategy you like, but jest works out of the box. Create test files in directories called `__tests__` or with the `.test.js` extension to have the files loaded by jest. See an example test [here](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/app/App.test.js). The [jest documentation](https://facebook.github.io/jest/docs/en/getting-started.html) is also a wonderful resource, as is the [React Native testing tutorial](https://facebook.github.io/jest/docs/en/tutorial-react-native.html).
 
+## UI Tests
+
+This repository uses appium to run UI tests. The tests live in `__device-tests__` and are written using Appium to run tests against simulators and real devices. To run these you'll need to check off a few things: 
+
+* For now you'll need run `yarn start`, and then either `yarn ios` or `yarn android` at least once before trying to run the tests on the respective platform
+* [Appium cli](https://github.com/appium/appium/blob/master/docs/en/about-appium/getting-started.md) installed and available globally, I'd also recommend using [appium doctor](https://github.com/appium/appium-doctor) to ensure all of appium's dependencies are good to go. You don't have to worry about starting the server yourself, the tests handle starting the server on port 4728, just be sure that the port is free or feel free to change the port number in the test file. 
+* For iOS a simulator should automatically launch but for Android you'll need to have an emulator fired up and running.
+
+After those are checked off to run the UI tests on ios run 
+
+`yarn test:ui:ios`
+
+and for android run, 
+
+`yarn test:ui:android`
+
+Alternatively if you're experiencing problems that seem to be related to the tests starting the appium server, for example errors that say `Connection Refused`, `Connection Reset` or `The requested environment is not available`. Sorry about that this is still a WIP, you can manually start the appium server via [appium desktop](https://github.com/appium/appium-desktop) or the cli, then change the port number in the tests while commenting out the `beforeAll` and `afterAll` block. 
```

Comment:
Syntactic nit: I see the `if you're...` but I think there's nothing for the "then" of this sentence.

Reply:
Hmm this so instead of a full stop then new sentence more of a 

> then you can manually start...

?

Reply:
Ah, I see. Sorry, my coder's brain keeps wanting to see the "then" in the same sentence as the "if" 😊

May I suggest this revision: `Note, you might experience problems...`, so there is no if anyway?

Reply:
Seems addressed by 8f6b24ef43eb613ff300b0ad228411e3fc40041a.